docs/jdbc-doris-catalog.md

```diff
@@ -0,0 +1,177 @@
+
+:::caution
+Gravitino saves some system information in schema and table comment, like `(From Gravitino, DO NOT EDIT: gravitino.v1.uid1078334182909406185)`, please don't change or remove this message.
+:::
+
+## Catalog
+
+### Catalog capabilities
+
+- Gravitino catalog corresponds to the Doris instance.
+- Supports metadata management of Doris (1.2.x).
+- Supports table index.
+- Supports [column default value](./manage-relational-metadata-using-gravitino.md#table-column-default-value).
+
+:::note
+Doris 2.x supports more advanced metadata features such as auto-increment column. We will be adding support for it in the future.
+:::
+
+### Catalog properties
+
+You can pass to a Doris data source any property that isn't defined by Gravitino by adding `gravitino.bypass` prefix as a catalog property. For example, catalog property `gravitino.bypass.maxWaitMillis` will pass `maxWaitMillis` to the data source property.
+
+Check the relevant data source configuration in [data source properties](https://commons.apache.org/proper/commons-dbcp/configuration.html)
+
+If you use a Doris catalog, you must provide `jdbc-url`, `jdbc-driver`, `jdbc-user` and `jdbc-password` to catalog properties.
+
+| Configuration item   | Description                                                                         | Default value | Required | Since Version |
+|----------------------|-------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `jdbc-url`           | JDBC URL for connecting to the database. For example, `jdbc:mysql://localhost:9030` | (none)        | Yes      | 0.5.0         |
+| `jdbc-driver`        | The driver of the JDBC connection. For example, `com.mysql.jdbc.Driver`.            | (none)        | Yes      | 0.5.0         |
+| `jdbc-user`          | The JDBC user name.                                                                 | (none)        | Yes      | 0.5.0         |
+| `jdbc-password`      | The JDBC password.                                                                  | (none)        | Yes      | 0.5.0         |
+| `jdbc.pool.min-size` | The minimum number of connections in the pool. `2` by default.                      | `2`           | No       | 0.5.0         |
+| `jdbc.pool.max-size` | The maximum number of connections in the pool. `10` by default.                     | `10`          | No       | 0.5.0         |
+
+:::caution
+You must download the corresponding JDBC driver to the `catalogs/jdbc-doris/libs` directory.
+:::
+
+### Catalog operations
+
+Refer to [Manage Relational Metadata Using Gravitino](./manage-relational-metadata-using-gravitino.md#catalog-operations) for more details.
+
+## Schema
+
+### Schema capabilities
+
+- Gravitino's schema concept corresponds to the Doris database.
+- Supports creating schema.
+- Supports dropping schema.
+
+### Schema properties
+
+- Support schema properties
```

Comment:
What kind of schema properties do we support currently?

Reply:
docs updated. `Support schema properties, includes Doris database properties and user-defined properties.`